### PR TITLE
disable prow branch protection for build repos

### PR DESCRIPTION
**Pull request by tnevrlka**
Disable prow branch protection in build-service and image-controller

### CODE REVIEW

**Review by Michkov — APPROVED** [review]

**Review by psturc — APPROVED** [review]
/lgtm

**Comment by tnevrlka** [comment]
typo

File: core-services/prow/02_config/redhat-appstudio/image-controler/_prowconfig.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+branch-protection:
```
Comment (psturc) [inline core-services/prow/02_config/redhat-appstudio/image-controler/_prowconfig.yaml:1]: hmm I think this is actually a wrong file

see https://github.com/openshift/release/blob/57dc33ea6e250f09f38fead1398fed97944c482c/core-services/prow/02_config/redhat-appstudio/image-controller/_prowconfig.yaml

**Review by psturc — CHANGES_REQUESTED** [review]

**Pull request merged**